composer.json

```diff
@@ -4,12 +4,20 @@
 		"cs:fix": "php-cs-fixer fix",
 		"lint": "find . -name \\*.php -not -path './vendor/*' -not -path './build/*' -print0 | xargs -0 -n1 php -l"
 	},
-    "require-dev": {
+	"require-dev": {
 		"nextcloud/coding-standard": "^1.0.0"
 	},
 	"config": {
+		"autoloader-suffix": "WorkflowPDFConverter",
+		"classmap-authoritative": true,
```

Comment:
you can skip this for better DX. The `-a` during packaging makes production fast

ref https://github.com/nextcloud/mail/pull/8020